openslides_backend/shared/exceptions.py

```diff
@@ -96,6 +101,8 @@ class PermissionDenied(ViewException):
 class MissingPermission(PermissionDenied):
     def __init__(
         self,
-        permission: Any,
+        permission: Union[
+            Permission, OrganisationManagementLevel, CommitteeManagementLevel
+        ],
     ) -> None:
         self.message = f"Missing permission: {permission}"
```

Comment:
Maybe we should change the message to include the type name of the permission, so that instead of `Missing permission: can_manage_users` it says `Missing OrganisationManagementLevel: can_manage_users`

Reply:
`type(..).__name__` doesn't give the right text. I use `isinstance` to get the permission category and set a string then.

Reply:
Why, what does it return? I'm not a fan of case distinctions there.

Reply:
It should return `Permission`. It returns `_Motion` as example.

Reply:
Ah, right. Well, I guess we can leave it like this then for now